app/views/admin/users/confirm_reactivate.html.haml

```diff
@@ -0,0 +1,25 @@
+= form_for [:admin, @user], url: edit_admin_user_path(@user), method: :post do |form|
+  .govuk-grid-row
+    .govuk-grid-column-full
+      %fieldset.govuk-fieldset
+        %legend.govuk-fieldset__legend.govuk-fieldset__legend--xl
+          %h1.govuk-fieldset__heading
+            Reactivate this User?
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %p
+        This user will be recreated on Auth0 and will be sent a new password
```

Comment:
GOV.UK content design principles prefer the [active voice](https://www.gov.uk/government/publications/govuk-content-principles-conventions-and-research-background/govuk-content-principles-conventions-and-research-background#writing-using-the-active-voice) – so we could go with "We will recreate this user on Auth0 and send them a new password"?

Reply:
I'll do that and maybe Christine can have a looksee